src/wp-content/themes/twentytwentythree/style.css

```
@@ -0,0 +1,16 @@
+/*
+Theme Name: Twenty Twenty-Three
+Theme URI: https://github.com/WordPress/twentytwentythree
```

Let's change this to the future Theme Directory location now so we don't forget before release. 

```suggestion
Theme URI: https://wordpress.org/themes/twentytwentythree
```